prime-router/metadata/hl7_mapping/datatypes/coding/MSG.yml

```diff
@@ -0,0 +1,13 @@
+elements:
+
+  - name: msg-message-code
+    hl7Spec: [ '%{hl7MSGField}-1' ]
+    value: [ '%resource.display.split("^")[0]' ]
```

Comment:
I love seeing the different ways you are able to avoid using extensions for the FHIR->HL7v2 conversion, its art!